### ISSUE TITLE
Correct first par in Simulation Infrastructure

### ISSUE THREAD

**Issue by bergtwvd** [AMSP-04/NETN-FOM#41]
To view the RTI as a distributed operating system is far-fetched:

`On the network, the RTI acts as a distributed operating system, providing a standard interface (API) to the federated systems.`

Reword this to:

`The RTI provides a standard interface (API) to the federated systems.`

Also remove `Internally the RTI uses distributed algorithms and network protocols to implement all HLA services` since this is already covered in the sentence before. 

Thus result is:

`NATO STANAG 4603 mandates the use of IEEE 1516-2010 standards on High-Level Architecture (HLA Evolved) for new M&S systems. The use of HLA in NETN federations is a fundamental design rule which impacts most parts of the NETN FAFD. A NETN federation includes a Run-Time Infrastructure (RTI) implementing the HLA services. The RTI provides a standard interface (API) to the federated systems. The NETN FAFD makes no recommendations concerning specific RTI implementations but requires complete and certified HLA RTI implementation.`

**Comment by lofstrandbjorn** [comment]
Implemented as suggested.
